testing/test_step_saving.py

```diff
@@ -0,0 +1,232 @@
+import os
+
+import numpy as np
+from joblib import dump
+from py._path.local import LocalPath
+
+from neuraxle.base import BaseStep, TruncableJoblibStepSaver, NonFittableMixin
+from neuraxle.checkpoints import PickleCheckpointStep
+from neuraxle.pipeline import ResumablePipeline
+
+OUTPUT = "OUTPUT"
+ROOT = 'ResumablePipeline'
+CHECKPOINT = 'checkpoint'
+SOME_STEP_2 = 'some_step2'
+PIPELINE_2 = 'pipeline2'
+SOME_STEP_1 = 'some_step1'
+SOME_STEP_3 = 'some_step3'
+
+EXPECTED_OUTPUTS = [0, 48, 96, 144, 192, 240, 288, 336, 384, 432]
+
+
+def create_some_step3_path(tmpdir, create_dir=False):
+    p = os.path.join(tmpdir, ROOT, PIPELINE_2, SOME_STEP_3, '{0}.joblib'.format(SOME_STEP_3))
+    if create_dir:
+        os.makedirs(os.path.join(tmpdir, ROOT, PIPELINE_2, SOME_STEP_3))
+    return p
```

Comment:
This method looks the same as the ones below. There is duplicated code here. Use a loop and a list or a parametrization list or something perhaps?